packages/ember/tests/helpers/link_to_test.js

```diff
@@ -1404,11 +1404,364 @@ QUnit.test("supplied QP properties can be bound (booleans)", function() {
 });
 
 QUnit.test("href updates when unsupplied controller QP props change", function() {
+  Ember.TEMPLATES.index = compile("{{#link-to (query-params foo='lol') id='the-link'}}Index{{/link-to}}");
+
+  bootApplication();
   var indexController = container.lookup('controller:index');
+
+  equal(Ember.$('#the-link').attr('href'), '/?foo=lol');
+  Ember.run(indexController, 'set', 'bar', 'BORF');
+  equal(Ember.$('#the-link').attr('href'), '/?bar=BORF&foo=lol');
+  Ember.run(indexController, 'set', 'foo', 'YEAH');
+  equal(Ember.$('#the-link').attr('href'), '/?bar=BORF&foo=lol');
```

Comment:
It's just copied from the existing link-to test, but yeah, both descriptions aren't communicative